tests/test_model.py

```diff
@@ -0,0 +1,13 @@
+from tutor.models.model import Model
+from tutor import db
+
+def test_add_new_model():
```

Comment:
I guess this is ok for now because you don't have any actual functionality yet, but this test is not actually testing your code, its testing the ORM. To make this kind of test useful, you would need to make it application-specific by for example:

1. Embed the queries on lines 9 and 12 into custom model methods so the test becomes a test for the query methods.
2. Test all the model properties so the test becomes a test that the model includes the properties that are needed for your application.

Reply:
There is only `name` property because that model was just for example and testing - no use in the app. I updated it as you asked.